### DISCUSSION TITLE
Can plots be moved from one farm to another (combined from two different keys)?

### DISCUSSION THREAD

**Discussion by jamieleben** [Chia-Network/chia-blockchain#2216] [Q&A]
Before I figured out remote harvesting I set up two different computers with two different keys and made plots under both keys.
Is it possible to combine the plots under one key (this would also be phrased as combining farms, right)?
Unless I misunderstand what's happening with plotting, I expect the answer is no.

I suppose Chai earned could be shared via pooling, but replotting the drives is no big deal.

If plots CAN be combined, I get to learn...
Just wanted to double check before I re-plot those drives.
Thanks!

**Comment by dorkmo** [comment]
plots are tied to the keys that created them, so you'll have to manage them separately.

**Reply by jamieleben** [reply]
Thank you! I figured that was the case. It's no big deal to replot those drives under the single key.
